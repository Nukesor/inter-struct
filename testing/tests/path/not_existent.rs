use inter_struct::InterStruct;

/// This shouldn't compile, as the path shows to an invalid location.
#[derive(InterStruct)]
#[into("crate::some_path::IntoStruct")]
pub struct FromStruct {
    pub normal: String,
}

pub struct IntoStruct {
    pub normal: String,
    pub optional: Option<String>,
}

fn main() {}
