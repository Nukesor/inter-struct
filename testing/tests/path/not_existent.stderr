error: Failed to open file "$WORKSPACE/target/tests/testing/src/some_path.rs": Os { code: 2, kind: NotFound, message: "No such file or directory" }
 --> tests/path/not_existent.rs:5:8
  |
5 | #[into("crate::some_path::IntoStruct")]
  |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
